launcher/actions/transformer_nest_to_tvb.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<settings>
+    <co_simulation_action>
+        <title>Nest to TVB Transformer</title>
+        <description>Nest to TVB Transformer</description>
+        <action>
+            <script>nest_to_tvb.py</script>
```

Comment:
These XML files are a good way to collect the components needed, here NEST to TVB. 

Maybe you can add a NOTE, that the names of the files will probably change.
E.g. 'test_input_nest_to_tvb.py' and 'test_receive_nest_to_tvb.py' are not named consistent. Should be input/output or send/receive.

Reply:
Understood, I totally agree on using mnemonic names. I will add the NOTE. I guess the .XML and the .py should have some name coincidence, which means that the python script that will perform the real processing should be referred as <script>transformer_nest_output_into_tvb_input.py</script>